korman/exporter/material.py

```diff
@@ -109,6 +109,8 @@ def _update(self, other):
 class MaterialConverter:
     def __init__(self, exporter):
         self._obj2mat = {}
+        self._bumpMats = {}
+        self._bumpLUT = None
```

Comment:
Since multiple objects might have bumpmapping, the idea was to reuse the same texture for all of them. Maybe there's a way to handle that at a resManager level when creating it initially?